Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -82,11 +85,15 @@ private void setUpMotorTeam(XCANTalon master) {
         master.configPeakOutputVoltage(12, -6);
 
         master.setProfile(0);
-        master.setF(.099);
         updateMotorPidValues(master);
         master.setControlMode(TalonControlMode.Speed);
-
         master.set(0);
+        
+        follower.setProfile(0);
+        follower.setControlMode(TalonControlMode.Follower);
+        follower.configNominalOutputVoltage(0,  -0);
+        follower.configPeakOutputVoltage(12, -6);
```

Comment:
Is the currently physically modeled config the same as the 2016 summer bot shooter?